translathon.qmd

```diff
@@ -2,12 +2,168 @@
 title: "Mini-translathon"
 ---
 
-Details about the format.
+## Overview
 
+Translathons are 2-hour events designed as a combination of *translation* with the ways and energy of a *hackathon* (a dedicated period of time where participants make contributions to selected FOSS projects).
+Translathons are live and synchronous: participants contribute, learn, and connect with others.
 
-The community call provided guidance on how to get started with translation contributions.
```

Comment:
Plan a community call to provide guidance on how to get started with translation contributions and different translations and localization efforts that exist for a human language in the community.